### DISCUSSION TITLE
Check value of Seedbox, herbsack, gembag

### DISCUSSION THREAD

**Discussion by kulershk** [runelite/runelite#15037] [Ideas]
So basically when you check whats inside it could show a value of it

![image](https://user-images.githubusercontent.com/26186873/39068207-2d73b32a-44e4-11e8-8780-1658e223085f.png)

**Comment by Gamer1120** [comment]
I vaguely remember there being a pull request for one of the options, but I can't find it. Maybe it does, maybe it doesn't exist.

**Comment by geheur** [comment]
Dude where's my stuff can show the value of the items in the seed box I think (Don't have one myself to test it).
